angelsindustries/prototypes/overrides/global-tech-base-packs.lua

```diff
@@ -127,8 +127,10 @@ if angelsmods.industries.tech then
   pack_replace("angels-titanium-smelting-1", "blue", "orange")
   pack_replace("angels-tungsten-smelting-1", "blue", "orange")
   pack_replace("angels-aluminium-smelting-2", "blue", "orange")
+  pack_replace("angels-cobalt-smelting-1", "blue", "orange")
   pack_replace("angels-cobalt-smelting-2", "blue", "orange")
   pack_replace("angels-glass-smelting-2", "blue", "orange")
+  pack_replace("angels-gold-smelting-1", "blue", "orange")
   pack_replace("angels-gold-smelting-2", "blue", "orange")
```

Comment:
I assume this meas tier 2 can move to blue (instead of orange?)

Reply:
ill need to double check it, but tier 1 was coming up as blue with the change, i think some of the products for 2's are actually used in the batteries required for blue...

Reply:
> i think some of the products for 2's are actually used in the batteries required for blue...

Then we should move those to tier 1 where possible?